codegen/model/types/parser.go

```diff
@@ -238,15 +238,26 @@ func (p *Parser) parseField(
 	}, nil
 }
 
-func (p *Parser) getFieldType(goType reflect.Type, protoField protoreflect.FieldDescriptor) (Type, error) {
+func (p *Parser) getFieldType(goType reflect.Type, protoField protoreflect.FieldDescriptor, mapType bool) (Type, error) {
 	// TODO: Since maps are not currently used, I didn't bother supporting them yet.
+	//if goType.Kind() == reflect.Map {
+	//	return nil, fmt.Errorf("map fields are not supported yet")
+	//}
```

Comment:
Probably can be removed now :)

Reply:
Done, the new code also supports defining only the key (by either not writing '|' or writing it as the last character when tagging).

Reply:
Done.